### PR TITLE
Add Quick Demo Section to README [5933]

### PR DESCRIPTION
**Pull request by AlvaroMolinillo**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -72,6 +72,23 @@ You can access the documentation online, which is hosted on [Read the Docs](http
...
+
+Then, you have to download the docker image file from https://eprosima.com/index.php/downloads-all
+
+Load the image and run it
+
+	$ docker load -i ubuntu-fast-rtps.tar
+	$ docker run -it ubuntu-fast-rtps
+
+You can run as many images as you want and check the communication between them
+
```
Comment (raquelalvarezbanos) [inline README.md]: Please copy this section from https://github.com/eProsima/Fast-RTPS/pull/584/files and make sure it is exactly the same.

**Comment by richiware** [comment]
Build status: * Linux [Build Status](http://jenkins.eprosima.com:8080/job/FastRTPS%20Manual%20Docs/2/)

**Comment by richiware** [comment]
Build status: [Build Status](http://jenkins.eprosima.com:8080/job/FastRTPS%20Manual%20Docs/2/)

**Comment by richiware** [comment]
Build status: [Build Status](http://jenkins.eprosima.com:8080/job/FastRTPS%20Manual%20Docs/5/)

**Comment by richiware** [comment]
Build status: [Build Status](http://jenkins.eprosima.com:8080/job/FastRTPS%20Docs%20Manual/11/)

**Pull request merged**